tldw_Server_API/app/core/DB_Management/Collections_DB.py

```
@@ -224,6 +264,22 @@ def _coerce_bool_flag(value: Any, *, postgres: bool) -> Any:
             return bool(value)
         return 1 if value else 0
 
+    def _sqlite_columns(self, table: str) -> set[str]:
+        if self.backend.backend_type != BackendType.SQLITE:
+            return set()
+        if table not in _SQLITE_PRAGMA_TABLES:
+            return set()
+        try:
+            result = self.backend.execute(f"PRAGMA table_info({table})", tuple())
+        except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)